forms-flow-web/src/components/Form/List.js

```diff
@@ -328,6 +301,37 @@ const List = React.memo((props) => {
       </span>
     );
   };
+  const searchInputBox = useRef("");
```

Comment:
Can we just use a state for search rather having a Ref . So value can be easily updated and use useEffect for change listen to call the api